crm_compassion/models/event_compassion.py

```diff
@@ -450,11 +473,18 @@ def _get_calendar_vals(self):
 
     @api.multi
     def allocate_children(self):
-        no_money_yield = float(
-            self.planned_sponsorships) / self.number_allocate_children
+        no_money_yield = float(self.planned_sponsorships)

```

Comment:
Previous version was correct, except that we need to multiply by 100 (number should be 0-100)